defaults/preferences/zotero.js

```diff
@@ -28,6 +28,7 @@ pref("extensions.zotero.downloadAssociatedFiles",true);
 pref("extensions.zotero.findPDFs.resolvers", '[]');
 pref("extensions.zotero.reportTranslationFailure",true);
 pref("extensions.zotero.automaticTags",true);
+pref("extensions.zotero.tagsFuzzyAutocomplete", false);
```

Comment:
Let's just optimize this as best as we can. No need to pref it (unless we start getting reports that it's causing problems).